{{cookiecutter.project_name}}/next_steps.md

```diff
@@ -0,0 +1,36 @@
+# Next steps
+
+## Put the generated files under version control
+
+Once your Python package is created, put it under [version
+control](https://guide.esciencecenter.nl/#/best_practices/version_control) using
+[git](http://git-scm.com/) and [GitHub](https://github.com/).
+
+```shell
+cd {{ cookiecutter.project_name }}
+git init
+git add --all
+git commit -m "first commit"
+git branch -M main
+git remote add origin {{ cookiecutter.repository }}
+```
+
+## Push the initial commit to a new repo on GitHub
+
+Go to
+[https://github.com/{{cookiecutter.github_organization}}?tab=repositories](https://github.com/{{cookiecutter.github_organization}}?tab=repositories)
```

Comment:
Could replace with link to create repo form directly for example https://github.com/organizations/NLeSC/repositories/new
```suggestion
[https://github.com/{{cookiecutter.github_organization}}/repositories/new](https://github.com/{{cookiecutter.github_organization}}/repositories/new)
```